src/services/checklistService.js

```diff
@@ -8,7 +8,7 @@ export {
     getChecklistById,
 }
 
-const BASE_URL = '/api/activities/'
+const BASE_URL = '/api/checklist/'
```

Comment:
@vzarate-94 this needs to be `/api/checklists/` to match the route in `server.js`